packages/flutter/test/cupertino/action_sheet_test.dart

```diff
@@ -1956,6 +1956,41 @@ void main() {
     );
   });
 
+  testWidgets('Hovering over Cupertino action sheet action updates to the passed cursor on desktop', (WidgetTester tester) async {
+    const SystemMouseCursor customCursor = SystemMouseCursors.click;
```

Comment:
Consider using another cursor to make the logic in the test independent of the "clickable on web" behavior.

Reply:
Still uses `.click`.

Reply:
This change, although small, is important because if this PR is regressed, and this test is run on web, it would still pass.

Reply:
oh so for web use a different one like maybe grab and for non web use something like click or forbidden?

Reply:
There doesn't need to be different cursors per platform. Just one cursor that is definitely not a default.

Correction: The test wouldn't pass either way because we're adding a property to a public API. It would only pass if the changes are regressed in a way that the property is no longer respected.

Reply:
understood.

So how would I handle it? Still not able to figure that portion out

Reply:
Replace `.click` with `.grab` or `.forbidden` etc.

Reply:
oh ok will update with that... 

apart from this change, others all look good or any changes still required?